tests/rules/javascript/test.no_eval.js

```diff
@@ -0,0 +1,101 @@
+import { VALIDATION_WARNING } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+
+// These rules were mostly copied and adapted from
+// https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-eval.js
+// Please make sure to keep them up-to-date and report upstream errors.
+describe('no_eval', () => {
+  var validCodes = [
+    'Eval(foo)',
+
+    // User-defined eval methods.
+    'window.eval("foo")',
+    'window.eval("foo")',
+    'window.noeval("foo")',
+    'function foo() { var eval = "foo"; window[eval]("foo") }',
+    'global.eval("foo")',
+    'global.eval("foo")',
```

Comment:
that was me failing adding the original tests since they were splitted between environments and I cleaned them up in a automated fashion and kinda failed 😆 I'll fix this.